templates/shared_services/gitea/terraform/mysql.tf

```diff
@@ -14,14 +14,15 @@ resource "azurerm_mysql_server" "gitea" {
   administrator_login_password      = random_password.password.result
   sku_name                          = "GP_Gen5_2"
   storage_mb                        = 5120
-  version                           = "8.0"
+  version                           = "8.0.21"
```

Comment:
I'm sure this is fine, but what's the reason for upgrading MySQL, and why this specific version?
This is newer than 8.0, but there are newer versions supported by Azure than this one.

It's also not clear whether this is a value value from a TF point of view. The TF docs suggest you can set this to 5.7 or 8.0 only. The docs also say that changing this value causes a new resource to be created, so are there migration steps needed for any of this?

Reply:
I agree...this is strange.

The portal offers 5.7 or 8.0:

![image](https://user-images.githubusercontent.com/1524451/171150992-7f4f84ae-8aa6-44e7-93a0-565cd63e7cf7.png)

... the swagger for the rest api specifies 8.0.21...

![image](https://user-images.githubusercontent.com/1524451/171151152-a883962f-0b1d-47fa-9ae6-ce176118247e.png)
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2021-05-01/mysql.json

... while the DOCS suggest that different versions are supported:

https://docs.microsoft.com/en-us/azure/mysql/single-server/concepts-supported-versions

![image](https://user-images.githubusercontent.com/1524451/171151475-f91fc6ee-c383-4831-8b55-3144d456849c.png)

Reply:
My concern is really that the current deployments specific 8.0. If we change this to anything else, will this cause an existing DB to be destroyed/recreated? If it does, migration needs to be addressed.

Can we tell tf lint to ignore this?

Reply:
Yes. Exactly what I did. I have also opened a [new issue](https://github.com/microsoft/AzureTRE/issues/1944) to remove it in the future once this bug is fixed.

Reply:
Perfect. Thanks @LizaShak :)